internal/pkg/service/common/etcdop/iterator/iterator_test.go

```diff
@@ -87,7 +87,7 @@ func TestIterator(t *testing.T) {
   ➡️  THEN:
   001 ➡️  GET ["some/prefix/", "some/prefix0")
 ✔️  TXN | succeeded: true | rev: %d
-➡️  GET ["some/prefix/foo004", "some/prefix0") | rev: %d
+➡️  GET ["some/prefix/foo004", "some/prefix0") | rev: %d | serializable
 ✔️  GET ["some/prefix/foo004", "some/prefix0") | rev: %d | count: 2
```

Comment:
Updated iterator tests.

`2+` page is loaded with locked revision and `serializable` flag. We don't need cluster consensus to read the specific revision.

From docs:
> By default, Range is linearizable; it reflects the current consensus of the cluster. For better performance and availability, in exchange for possible stale reads, a serializable range request is served locally without needing to reach consensus with other nodes in the cluster.

Reply:
:telephone_receiver: